packages/darma-vt/package.py

```diff
@@ -155,6 +155,7 @@ class DarmaVt(CMakePackage):
     depends_on("mpi")
     depends_on("darma-detector")
     depends_on("darma-checkpoint")
+    depends_on("fmt")
```

Comment:
Need to constrain to vt develop (or in general 1.5 and later when that is released)